build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
yes you are correct there is no one way which will make the entire thing consistent. The main thing i wanted to avoid is duplicating the copy task for cross build.

As you download the debugger binaries once on the build machine and copy it with all the tests on to a run machine. But it could be argued that the second copy is an implementation detail of the deployment mechanism. Any way i am fine with this change